src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/orderedproperties/Example1.properties

```diff
@@ -0,0 +1,13 @@
+#xml
+#<module name="Checker">
+#<module name="OrderedProperties"/>
+#</module>
+
```

Comment:
@lmgyuan , Please go through the issue description again. Refer successful PRs for better understanding.
You will have to make changes here.